### PR TITLE
Feature coin table

### PR DESCRIPTION
**Pull request by Chase-T-White**
1. Created dark/light mode theme
2. Created Coin table with infinite scroll
3. Moved some files around

### CODE REVIEW

File: app/ui/home/CryptoCoinTable.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  }, [dispatch, pageNumber]);
+
+  const changeSort = (category: string) => {
+    setSortRotation(
+      sortRotation === "" || category !== sortCategory
+        ? "asc"
+        : sortRotation === "asc"
+        ? "desc"
+        : ""
+    );
```
Comment (yoloOnTheBattlefield) [inline app/ui/home/CryptoCoinTable.tsx]: no nested ternaries

File: app/ui/home/CryptoCoinTable.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                if (typeof a[sortCategory] === "number") {
+                  return a[sortCategory] > b[sortCategory] ? -1 : 1;
+                }
+                return b[sortCategory]
+                  .toString()
+                  .localeCompare(a[sortCategory]);
+              } else {
+                return a - b;
+              }
+            })
```
Comment (yoloOnTheBattlefield) [inline app/ui/home/CryptoCoinTable.tsx:103]: extrapolate this in a function

File: app/ui/navbar/buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        data-hide-on-theme="dark"
+      />
+      <Image
+        src="/images/Dark.svg"
+        alt="Dark icon"
+        width={20}
+        height={20}
+        data-hide-on-theme="light"
+      />
+      {/* <Image src="/images/Light.svg" alt="Light icon" width={20} height={20} /> */}
```
Comment (yoloOnTheBattlefield) [inline app/ui/navbar/buttons.tsx]: remove this

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Comment by Chase-T-White** [comment]
Issues have been resolved

**Pull request merged**